app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,38 @@
+@import 'bootstrap/dist/css/bootstrap.min';
+
+body {
+  position: fixed;
+  background-color: pink;
```

Comment:
Would prefer to use a hex color; or for readability, use variables that are assigned to hex codes. 

For example:
```scss
$pink: #e056fd;

.some-class { 
   background-color: $pink;
}
```